.snapcraft/snap/hooks/configure

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# Obtain siteurl value
+siteurl="$(snapctl get site-url)"
+# Validate it
+#siteurl_regex='^https?:\/\/([0-9A-Za-z\.-]+){1,}(\.[a-z\.]{2,6})?([\/\da-z\.-]+)?$'
+siteurl_regex='^https?:\/\/([0-9A-Za-z\.-]+){1,}(\.[a-z\.]{2,6})?$'
```

Comment:
We should allow port on this also.  So something like http://mysite:3000 should pass.